keanu-project/src/test/java/io/improbable/keanu/vertices/ProxyVertexTest.java

```diff
@@ -0,0 +1,13 @@
+package io.improbable.keanu.vertices;
+
+import io.improbable.keanu.vertices.intgr.nonprobabilistic.IntegerProxyVertex;
+import org.junit.Test;
+
+public class ProxyVertexTest {
+
+    @Test(expected = RuntimeException.class)
```

Comment:
please use a `@Rule ExpectedException` so that you assert on the message too

There's examples in other test files that you can copy.